build.sbt

```diff
@@ -162,6 +162,12 @@ lazy val codegenSbt = project
   .in(file("codegen-sbt"))
   .settings(name := "caliban-codegen-sbt")
   .settings(commonSettings)
+  .enablePlugins(BuildInfoPlugin)
```

Comment:
Was this actually used for anything?

Reply:
Yes, for the caches. When the user will update Caliban version, the caches will be invalidated which will re-trigger the generation of the code. I thought it could be a good idea.

Reply:
😅 `Show full diff` on the main file was still not clicked, it didn't seem used. Good idea, certainly!